plugins/InspectionGadgets/InspectionGadgetsAnalysis/resources/messages/InspectionGadgetsBundle.properties

```diff
@@ -1657,6 +1657,8 @@ if.may.be.factorized.problem.descriptor=<code>#ref</code> can be factorized #loc
 if.may.be.factorized.quickfix=Replace with factorized expression
 redundant.string.format.call.display.name=Redundant call to 'String.format()'
 redundant.call.problem.descriptor=Redundant call to <code>#ref()</code> #loc
+inspection.simplifiable.compare.java.time.display.name=Comparison with compareTo can be simplified
```

Comment:
How about: `Expression with ''java.time'' <code>#ref()</code> call can be simplified`.
#ref will expand to the highlighted area (which will have to be modified a bit as well)

Reply:
Updated display name and problem descriptor. Thank you.